test/gasBenchmarks/deployments.js

```diff
@@ -24,9 +24,19 @@ contract('Gas Benchmark - [contract deployments]', async (accounts) => {
     it('Should deploy all contracts and print benchmarks', async () => {
        let accessControlInstance = await AccessControlSegregatorContract.new(
             [
-                "adminPauseTransfers", "adminUnpauseTransfers", "adminSetResource", "adminSetGenericResource", "adminSetBurnable",
-                "adminSetDepositNonce", "adminSetForwarder", "adminChangeAccessControl", "adminChangeFeeHandler", "adminWithdraw",
-                "startKeygen", "endKeygen", "refreshKey",
+                "0x80ae1c28", // adminPauseTransfersG
```

Comment:
Typo and maybe export the signatures array from `helpers.js`

Reply:
It is exported via `deployBridge` function`